src/main/java/org/myrobotlab/service/HtmlFilter.java

```diff
@@ -111,7 +111,9 @@ public void setStripHtml(boolean stripHtml) {
   public static String stripHtml(String text) {
     // TODO: something fancier but this works for now.
     String cleanText = text.replaceAll("\\<.*?\\>", " ");
-    cleanText = cleanText.replaceAll("  ", " ");
+    cleanText = cleanText.replaceAll("\\n", " ");
+    cleanText = cleanText.replaceAll("\\r", " ");
+    cleanText = cleanText.replaceAll("\\s{2,}", " ");

```

Comment:
Please revert any changes to this service.  If you want to improve the html stripping capabilities, it should be done as an isolated pull request and the change should be to use an official library to do this stripping.